starknet/src/utils/types.cairo

```diff
@@ -37,7 +36,7 @@ enum Choice {
     Abstain: (),
 }
 
-#[derive(Copy, Drop, Serde, PartialEq)]
+#[derive(Drop, Serde, PartialEq, Copy, starknet::Store)]
```

Comment:
why `starknet::Store` and not `Store` ?

Reply:
for some reason it wouldnt compile with `Store`

Reply:
not sure why

Reply:
weird... I don't see why it wouldn't but ok !